test/1_FantomAuctionv2.test.js

```diff
@@ -1238,12 +1238,17 @@ contract('FantomAuction', async function () {
     });
 
     // Test case **ID: A**::
-    it('092) `bidder` cannot withdraw a bid before auction ends', async function () {
+    it('092) `bidder` can successfully withdraw a bid before auction ends IF bid is below reserve price', async function () {
       await expect(
         fantomauction.connect(bidder).withdrawBid(mockerc721.address, TWO)
-      ).to.be.revertedWith(
-        'can withdraw only after 12 hours (after auction ended)'
-      );
```

Comment:
Please keep the original test `092) `bidder` cannot withdraw a bid before auction ends` - it still should check it is not possible to withdraw a bid before the end of the auction or when the price is bellow the reserve.
For new test please use new test IDs.